spec/models/talentlms/facade_spec.rb

```diff
@@ -73,6 +73,64 @@
     end
   end
 
+  describe 'active_user?' do
+    it 'returns true if user is active and no api_response data is included' do
+      expect(lms.active_user?(config, lms_login)).to be_truthy
+    end
+
+    it 'returns true if user is active in provided api response' do
+      api_stub_response = lms.get_lms_account_data(config, lms_login)
+      expect(lms.active_user?(nil, nil, api_stub_response)).to be_truthy
+    end
+
+    it 'returns false if user is not active' do
+      stub_so_api_user_is_inactive(config: config)
+      expect(lms.active_user?(config, lms_login)).to be_falsey
+    end
+
+    it 'returns false if user is active in provided api response' do
```

Comment:
Is this just missing a not?
```suggestion
    it 'returns false if user is not active in provided api response' do
```

Reply:
yes! thanks for catching that. I'll include it in the next commit